### DISCUSSION TITLE
same as last time--weather.com video won't play

### DISCUSSION THREAD

**Discussion by cmcjsrb** [uBlockOrigin/uAssets#15527] [General]
11-3  --weather.com at it again--with u-block enabled the video's won't play--did a purge and a update to latest--no help--has to be turned off and then you watch every add--anyone help--this time it has gone on for more than 2 days--even with me doing a cache purge and then update.. I just give up  I think I should just delete the site from my bookmarks.

**Comment by mapx-** [comment]
does this work for you ?

`@@||micro.rubiconproject.com/prebid/dynamic/$script,domain=weather.com`

**Comment by cmcjsrb** [comment]
that lets them play--with one gotcha--I have to let the main one play and then let it run through all the videos 1 by 1 on the right side of the main box.  if I click on another video before the one playing is over, nothing happens, before if I clicked on one before it was over because I didn't want to see any more of it--the click would change the video to the one I clicked on.  not now.  how come this happens on my comp. and nobody else seems to have a problem.  I don't get what's going on with this site, I don't have a problem with u-tube changing vids or on any other site I have booked marked.  thanks for you help

**Comment by mapx-** [comment]
This other exception should fix your second issue

`@@||ssl.p.jwpcdn.com/player/*/googima.js$script,domain=weather.com`

**Reply by cmcjsrb** [reply]
thanks again--as usual--really appreciate what you do
